jsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
+    "jsx": "preserve",
     "paths": {
-      "#/*": ["./src/*"]
+      "@/*": ["./src/*"]
```

Comment:
Why is it necessary? It changes so many files, which is painful when merging from master.

Reply:
Because there are webpack loaders only supporting `@` but not `#`. Some of them were confirmed as bugs and fixed but I've encountered several times. So I want to follow the popular tools like vite, to avoid such cases for good.

Reply:
I think we can make a release this weekend to fix the remaining bugs, and then merge this MR to get rid of the alias issue.

Reply:
I wonder if we can change to @ in master before merging so that we don't feel pressured to merge?

Reply:
I mean "pressured to merge this PR earlier" :-)